grpc/core/pom.xml

```diff
@@ -48,7 +48,17 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-protobuf</artifactId>
     </dependency>
-
+    <dependency>
```

Comment:
This uses 2 spaces indent, instead of 4.
In fact, it's not only in this PR, looking at the the grpc pom files in the master branch I see mixed indentations in some of them.

Reply:
I've fixed this and also fixed the other gRPC `pom.xml` files that were using two spaces.